spec/decorators/has_hint_option_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.describe HasHintOption do
+  subject { described_class.new(base_input) }
+  let(:object) { instance_double(GenericAssetForm) }
+  let(:base_input) { instance_double(HydraEditor::Fields::Input) }
+  let(:validators) { {} }
+  let(:property) { :lcsubject }
+  let(:nil_message) {[nil].compact}
+  let(:validator_message) {["Here is a message"]}
```

Comment:
You don't need validator_message or validator_messages anymore, since all you care about is the #property_hint interface.